inference-engine/samples/benchmark_app/remote_blobs_filling.hpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#if defined(OPENCL_FOUND)
+#    include <gpu/gpu_context_api_ocl.hpp>
+#endif
+
+#include <inference_engine.hpp>
+
+#include "infer_request_wrap.hpp"
+#include "utils.hpp"
+
+namespace GPU {
+
+#if defined(OPENCL_FOUND)
+struct OpenCL {
+    cl::Context _context;
+    cl::Device _device;
+    cl::CommandQueue _queue;
+
+    explicit OpenCL(std::shared_ptr<std::vector<cl_context_properties>> media_api_context_properties = nullptr) {
+        // get Intel iGPU OCL device, create context and queue
+        {
+            const unsigned int refVendorID = 0x8086;
+            cl_uint n = 0;
+            cl_int err = clGetPlatformIDs(0, NULL, &n);
```

Comment:
why did you use c api, instead of c++ API? I think C++ api will make it simpler.

Reply:
I just reused an existing helper wrapper for OpenCL to save time, but I can modify this part if it's critical

Reply:
c++ headers are distributed in separate package (on ubuntu at least), and some user may not have it installed on the machine. So lets use C API to cover more users, as benchmark_app is supposed to be built on user machine even for binary OV distribution